Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerMetrics.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+namespace Multiplayer
+{
+    enum MultiplayerGroupIds
+    {
+        MultiplayerGroup_Networking = 101
+    };
+
+    enum MultiplayerStatIds
+    {
+        MultiplayerStat_EntityCount = 1001,
```

Comment:
nit: consider adding comments as to what these are.

Reply:
Done